README.md

```diff
@@ -1,2 +1,48 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Capstone1-Microverse-Conference-Page
-This is a the first Microverse capstone project for HTML/CSS &amp; Javascript
+
+> This is a the first Microverse capstone project for HTML/CSS &amp; Javascript.
+
+## Built With
+
+- HTML, CSS & Javascript
+- Flexbox
+
+## Live Demo
+
+[Loom Video Demonstration](https://www.loom.com/share/3ed80fcda3624198bee52c14edca8747)
+
+[Live Demo Link](https://jonahkayizzi.github.io/Capstone1-Microverse-Conference-Page/)
+
```

Comment:
- Kindly make sure those link working properly, when I click those links it shows this page below, fix that, please

Reply:
Hello @MaganaAsiati , Thanks for the observation.

This is because I have not merged the pull request yet. I will deactivate the link for now until I merge